Gemfile

```diff
@@ -35,7 +35,6 @@ end
 group :test do
   gem 'rails-controller-testing', '~>1.0.1'
   gem 'capybara'
-  gem 'shoulda-matchers', '~> 2.8', require: false
```

Comment:
👍 They were added to reinforce TDD practices, but happy to remove them, 
as they are not testing behaviour